GliaWidgets/ViewController/Survey/SurveyViewController.View.swift

```diff
@@ -134,20 +149,27 @@ extension Survey {
 extension NSAttributedString {
     static func withRequiredSymbol(
         foregroundColor: UIColor,
-        fontSize: CGFloat,
-        fontWeight: CGFloat,
+        font: UIFont,
         isRequired: Bool,
         text: String
     ) -> NSAttributedString {
         let mutableString = NSMutableAttributedString(
             string: text,
             attributes: [
                 .foregroundColor: foregroundColor,
-                    .font: UIFont.systemFont(ofSize: fontSize, weight: .init(fontWeight))
+                .font: font
             ]
         )
         if isRequired {
-            mutableString.append(.init(string: " *", attributes: [.foregroundColor: UIColor.red]))
+            mutableString.append(
+                .init(
+                    string: " *",
```

Comment:
Should this asterisk belong to localisation string?

Reply:
I guess yes, along with obelix)